simulations/vip-converter/vip-converter-mainnet/simulations.ts

```diff
@@ -0,0 +1,120 @@
+import { expect } from "chai";
+import { Contract, Signer } from "ethers";
+import { ethers } from "hardhat";
+
+import { expectEvents, initMainnetUser } from "../../../src/utils";
+import { forking, testVip } from "../../../src/vip-framework";
+// imported addresses from converter vip
+import {
+  BTCBPrimeConverterTokenOuts,
+  BaseAssets,
+  ETHPrimeConverterTokenOuts,
+  RiskFundConverterTokenOuts,
+  USDCPrimeConverterTokenOuts,
+  USDTPrimeConverterTokenOuts,
+  XVSVaultConverterTokenOuts,
+  converters,
+} from "../../../vips/vip-248/vip-248/Addresses";
+import {
+  LIQUIDATOR,
+  NEW_RISK_FUND_CONVERTER_IMP,
+  NEW_SINGLE_TOKEN_CONVERTER_IMP,
+  PROXY_ADMIN,
+  PSR,
+  RISK_FUND_CONVERTER_PROXY,
+  SINGLE_TOKEN_CONVERTER_BEACON,
+  vipConverter,
+} from "../../../vips/vip-converter/bscmainnet";
+import BEACON_ABI from "../abi/Beacon.json";
+import DEFAULT_PROXY_ADMIN_ABI from "../abi/DefaultProxyAdmin.json";
+import LIQUIDATOR_ABI from "../abi/Liquidator.json";
+import PROTOCOL_SHARE_RESERVE_ABI from "../abi/ProtocolShareReserve.json";
+import SINGLE_TOKEN_CONVERTER_ABI from "../abi/SingleTokenConverter.json";
+import TRANSPARENT_PROXY_ABI from "../abi/TransparentProxyAbi.json";
+
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+
+const COMPTROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
+const ASSETS = [
```

Comment:
I would add more assets to the simulation. I would add every asset listed [here](https://docs-v4.venus.io/deployed-contracts/core-pool#underlying-tokens), except BUSD TRXOLD, TUSDOLD, UST, LUNA, VAI, VRT

Reply:
Done